libs/user/website/src/lib/image-upload/image-upload.tsx

```diff
@@ -1,69 +1,88 @@
-import {IconButton, styled} from '@mui/material'
-import {ChangeEvent, useId, useRef} from 'react'
+import {css, styled} from '@mui/material'
+import {BuilderImageUploadProps, useWebsiteBuilder} from '@wepublish/website/builder'
+import {useId, useRef} from 'react'
 import {MdDelete, MdEdit, MdOutlineUploadFile} from 'react-icons/md'
-import placeholderImg from './placeholder.svg'
+import {ReactComponent as PlaceholderImage} from './placeholder.svg'
 
-export type ImageUploadProps = {
-  image?: {url?: string | null | undefined}
-  onUpload: (image: ChangeEvent<HTMLInputElement> | null) => void
-  className?: string
-}
+export const ImageUploadWrapper = styled('div')`
+  display: grid;
+  gap: ${({theme}) => theme.spacing(2)};
+  grid-template-columns: max-content 1fr;
+  align-items: center;
+`
 
-export const ImageWrapper = styled('div')`
+export const ImageUploadImageWrapper = styled('div')`
```

Comment:
Renamed things for no conflicts in the future as everything gets re-exported in website